Does anyone else think that it's strange that fs.exists doesn't have err as its first parameter?